resources/carrier/en/44.txt

```diff
@@ -13,9 +13,108 @@
 # limitations under the License.
 
 # Carrier details updated from http://www.ofcom.org.uk/static/numbering/S7.xls
-# Carrier names have been updated to friendlier trading/brand names
-# The mapping of company names can be found in the project below.
-# Generated on 2017-07-14 via https://github.com/giggsey/GBCarrierGenerator
+# Generated on 2017-07-14.
+
+# Carrier names have been updated to friendlier trading/brand names.
+# The mapping of company names can be found in the project below:
```

Comment:
you don't mention the project now. Also I think it's fair to still reference your project in the way you did at line 18.